scripts/select_appointments.sql

```diff
@@ -1 +1 @@
--- retrieve appointment information
+SELECT FROM appointments
```

Comment:
Nope. `SELECT` is the method for selecting what pieces of data we want, which correlates to what columns we have in the table. You can't call it without arguments. 

Take a look at this solution:
```sql
SELECT p.surname, a.date, a.time
  FROM patients p
  INNER JOIN appointments a ON a.patient_id = p.id
  INNER JOIN doctors d ON a.doctor_id = d.id
    WHERE d.surname = 'Hubbard';

SELECT d.surname, a.date, a.time
  FROM doctors d
  INNER JOIN appointments a ON a.doctor_id = d.id
  INNER JOIN patients p ON a.patient_id = p.id
    WHERE p.surname = 'Wilcox';
```